.azure_pipelines/job_templates/build-verify-publish-template-unix.yml

```diff
@@ -0,0 +1,125 @@
+
+      - ${{ if eq(parameters.is_daily_build, 'True') }}:
+        # Run all unit tests to demonstrate successful PyTorch build
+        - bash: python test/run_test.py --continue-through-error --exclude-jit-executor --shard 1 2 --verbose
+          displayName: Run all unit tests
+
+      # Run ComponentGovernance
+      - task: ComponentGovernanceComponentDetection@0
+        inputs:
+          scanType: 'Register'
+          verbosity: 'Verbose'
+          alertWarningLevel: 'High'
+
+    # Build PyTorch and produce artifacts for verification stage
+    - ${{ if eq(parameters.is_official_build, 'True') }}:
+      # Build PyTorch from source in install mode and exclude test binaries
+      - bash: python setup.py install
+        displayName: Build PyTorch from source without test binaries
+
+      # Package PyTorch Wheel
+      - bash: python setup.py bdist_wheel
+        displayName: Package PyTorch Wheel
+
+      # Publish PyTorch Wheel
+      - task: PublishPipelineArtifact@1
+        inputs:
+          targetPath: $(Build.SourcesDirectory)/dist/
+          artifactName: Build_$(Build.BuildNumber)_$(configuration)
+        displayName: Publish PyTorch Wheel to Pipeline Artifacts
+
+  # Verification stage
+  - ${{ if eq(parameters.verify_stage, 'True') }}:
+    # Download PyTorch Wheel
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifact: Build_$(Build.BuildNumber)_$(configuration)
+        path: $(Build.SourcesDirectory)/verify
+      displayName: Download PyTorch Wheel
+
+    # Install PyTorch Wheel on Windows
+    - bash: python -m pip install $(Build.SourcesDirectory)/verify/torch*linux*.whl
+      displayName: Install PyTorch Wheel
+
+    # Ensure PyTorch installed correctly from produced wheel
+    - bash: |
+        cd $(Build.SourcesDirectory)/verify
+        python -c "import torch; print('Installed Torch version: ' + torch.__version__)"
+      displayName: Check PyTorch correctly installed from wheel
+
+  # Publishing stage
+  - ${{ if eq(parameters.publish_stage, 'True') }}:
+    # Download PyTorch Wheel
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifact: Build_$(Build.BuildNumber)_$(configuration)
+        path: $(Build.SourcesDirectory)/publish
+      displayName: Download PyTorch Wheel
+
+    # Publish wheel to Azure Artifacts
+    - bash: |
+        export TORCH_VERSION=$(head -c 5 ./version.txt)
+        export BUILD_DATE=$(date '+%Y%m%d')
+        export LAST_COMMIT=$(git rev-parse --short HEAD)
+        cd $(Build.SourcesDirectory)/publish
+        export TORCH_WHEEL=$(echo torch*linux*whl)
+        az extension add -n azure-devops
+        echo $ADOTOKEN | az devops login
+        az artifacts universal publish --organization "https://aiinfra.visualstudio.com/" --project "0628bce4-2d33-499e-bac5-530e12db160f" --scope project --feed "PyTorch" --name $TORCH_WHEEL --description "PyTorch Official Build Artifact" --version $TORCH_VERSION-$BUILD_DATE-$LAST_COMMIT --path .
```

Comment:
Hmm, why organization and project need to be hardcoded here, rather than be a pipeline parameters?

Reply:
They don't have to be hardcoded there, I will change them and the ones for the Windows build to be pipeline parameters.